Common/cpp/ReanimatedRuntime/WorkletRuntimeDecorator.cpp

```diff
@@ -125,6 +140,13 @@ void WorkletRuntimeDecorator::decorate(
             rt, synchronizedDataHolderRef, newData);
       });
 
+  jsi_utils::installJsiFunction(
+      rt,
+      "_getDataSynchronously",
+      [](jsi::Runtime &rt, const jsi::Value &synchronizedDataHolderRef) {
+        return reanimated::getDataSynchronously(rt, synchronizedDataHolderRef);
+      });
+
```

Comment:
Adding `_getDataSynchronously` to worklet runtime could be moved to a separate PR.

Reply:
Where `_getDataSynchronously` is used? 🤔

Reply:
I'm using it in react-native-vision-camera. Currently, there seems to be some problem with copying shared values (or rather mutables) to worklet runtimes as they become regular JS objects (e.g. `{value: 42}`) and thus lose reactivity. The workaround for me was to use `SynchronousDataHolder` which is a `jsi::HostObject` so it can be passed to arbitrary worklet runtime. In there, I'm using `_getDataSynchronously` to read the current value and `_updateDataSynchronously` to update it. Nice thing about this is that it's indeed synchronous and uses mutexes instead of being asynchronous. Since we already inject `_updateDataSynchronously`, I thought there's no reason not to inject `_getDataSynchronously` as well.